lib/smart_answer_flows/business-coronavirus-support-finder/outcomes/results.erb

```diff
@@ -13,21 +13,23 @@
     $CTA
     ### Coronavirus Job Retention Scheme (furlough scheme)
 
-    If you've had to ask your employees to stop working or work less because of coronavirus (put them on 'furlough') you can get support to pay their wages for the hours they do not work.
+    If you've had to ask your employees to stop working or work less because of coronavirus (put them on 'furlough') you can get support to pay their wages for the hours they do not work. The scheme has been extended until the end of September 2021.
 
-    The government will pay 80% of employees’ usual wages for hours they do not work, up to a maximum of £2,500 per month.
+    The government will pay:
 
-    The scheme is open until 30 April 2021.
+    - 80% of employees’ usual wages for hours they do not work, up to a maximum of £2,500 per month, until the end of June
+    - 70% of employees' usual wages for hours they do not work, up to a maximum of £2187.50 per month, until the end of July
+    - 60% of employees' usual wages for hours they do not work, up to a maximum of £1875 per month, until the end of August and September
 
-    You must submit your claim by:
+    The employees must receive 80% of their usual wages up to a maximum of £2500 per month until the end of September 2021.
+    From July 2021 to September 2021, you will have to make up the difference between the government payment and what the employee should receive.
 
-    - 15 March 2021, for eligible employees on furlough in February 2021
 
+    You must submit your claims by 11.59pm 14 calendar days after the month you’re claiming for. If this time falls on the weekend or a bank holiday then claims should be submitted on the next working day.
+    For example you must submit your claim by:
```

Comment:
think we need to add a line break here so the markdown list works

Reply:
Thanks